LEAF_Nexus/js/orgchartForm.js

```diff
@@ -100,11 +100,13 @@ orgchartForm.prototype.getForm = function (UID, categoryID, indicatorID) {
         success: function (response) {
           dialog.setContent(response);
           $("input:visible:first, select:visible:first").focus();
-          $("input:visible:first, select:visible:first").keypress(function (
+          $("input:visible:first, select:visible:first").on("keydown", function (
             event
           ) {
             if (event.which == 13) {
-              $("#" + dialog.btnSaveID).trigger("click");
+              setTimeout(() => { //timeout needed for ind 2 chosen dropdown
```

Comment:
Not sure what else might need to be reworked, but timeouts should not be used as they are less deterministic than typical event handlers.

Reply:
I found another (and I agree better) solution to this which should result in the behavior being what it had been previously.  If the chosen dropbox is open, the first 'enter' will just close it.

I had looked at the orgchart indicator formats and the position case looked like the only one that had a field with a dropdown - if this is not true I can also add this elsewhere.